connector/fyle_integrations_platform_connector/fyle_integrations_platform_connector.py

```diff
@@ -88,9 +88,12 @@ def set_workspace_id(self):
         self.departments.set_workspace_id(self.workspace_id)
         self.dependent_fields.set_workspace_id(self.workspace_id)
 
-    def import_fyle_dimensions(self, import_taxes: bool = False):
+    def import_fyle_dimensions(self, import_taxes: bool = False, import_dependent_fields: bool = False):
```

Comment:
doing this optionally so we can use this version everywhere without updating the client